empress/support_files/js/color-options-handler.js

```diff
@@ -0,0 +1,273 @@
+        var colormapP = this.container.appendChild(document.createElement("p"));
+        var colormapLbl = colormapP.appendChild(
+            document.createElement("label")
+        );
+        colormapLbl.innerText = "Color Map";
+        var colormapSC = colormapP.appendChild(document.createElement("label"));
+        colormapSC.classList.add("select-container");
+        this.colormapSelector = document.createElement("select");
+        Colorer.addColorsToSelect(this.colormapSelector);
+        colormapSC.appendChild(this.colormapSelector);
+        this.colormapSelector.id =
+            "color-options-handler-" + this.uniqueNum + "-colormap-select";
+        colormapLbl.setAttribute("for", this.colormapSelector.id);
+
+        // Add a row for choosing whether the color scale should
+        // be reversed
+        var reverseColormapP = this.container.appendChild(
+            document.createElement("p")
+        );
+        var reverseColormapLbl = reverseColormapP.appendChild(
+            document.createElement("label")
+        );
+        reverseColormapLbl.innerText = "Reverse Color Map";
+        this.reverseColormapCheckbox = reverseColormapP.appendChild(
+            document.createElement("input")
+        );
+        this.reverseColormapCheckbox.id =
+            "color-options-hanlder-" + this.uniqueNum + "-reverse-chk";
+        this.reverseColormapCheckbox.setAttribute("type", "checkbox");
+        this.reverseColormapCheckbox.classList.add("empress-input");
+        reverseColormapLbl.setAttribute("for", this.reverseColormapCheckbox.id);
+
+        var scope = this;
+        var notify = function () {
+            var options = scope.getOptions();
+            _.each(scope.observers, function (obs) {
+                obs.colorOptionsUpdate(options);
+            });
+        };
+
+        if (this.enableContinuousColoring) {
+            // add continuous values checkbox
+            var continuousValP = this.container.appendChild(
+                document.createElement("p")
+            );
+            var continuousValLbl = continuousValP.appendChild(
+                document.createElement("label")
+            );
+            continuousValLbl.innerText = "Continuous values?";
+            this.continuousValCheckbox = continuousValP.appendChild(
+                document.createElement("input")
+            );
+            this.continuousValCheckbox.id =
+                "color-options-hanlder-" + this.uniqueNum + "-continuous-chk";
+            this.continuousValCheckbox.setAttribute("type", "checkbox");
+            this.continuousValCheckbox.classList.add("empress-input");
+            continuousValLbl.setAttribute("for", this.continuousValCheckbox.id);
+            // Hide the "Continuous values?" stuff by default, since the default
+            // colormap is discrete
+            continuousValP.classList.add("hidden");
+
+            // add continuous values min/middle/max inputs
```

Comment:
```suggestion
            // add continuous values min/max inputs
```

Since we currently don't use a `middle` value input option (I guess that might be useful eventually, but not needed now)

Reply:
I've thought about this, yes it would *nice* to have a middle value -- but the workaround is to just have the colormap be symmetrical (i.e. -5, 5 implies centering around 0).  And let's be honest, most diverging colorbars are symmetrical, so having a center may not even be necessary.